src/SourceLink.GitWeb/buildMultiTargeting/SourceLink.GitWeb.targets

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
```

Comment:
The file name is missing `Microsoft.` prefix. That's probably why the build is failing.

Reply:
Done